File: tests/nonsmoke/functional/CompileTests/OpenMP_tests/section1.c
#include <omp.h>
#include <stdio.h>
int main()
{
#pragma omp parallel
  {
#pragma omp single
    printf ("This is from the single directive\n");
#pragma omp sections nowait
    {
#pragma omp section
      {
        printf("hello from section 1\n");
      }
#pragma omp section
      {
        printf("hello from section 2\n");
      }
#pragma omp section
      {
        printf("hello from section 3\n");
      }
    } /* end of sections */
  }/* end of parallel */
  return 0;
}

File: tests/nonsmoke/functional/CompileTests/OpenMP_tests/fortran/section1.f
      PROGRAM HELLO
!$OMP PARALLEL
!$omp single
      PRINT *, 'Hello World from single'
!$omp end single
!$omp sections 
!$omp section
      PRINT *, 'Hello World from section 1'
!$omp section
      PRINT *, 'Hello World from section 2'
!$omp end sections nowait
c$omp end parallel 
      PRINT *, 'the end'
      END

